src/components/base/link/Link.js

```diff
@@ -341,16 +342,25 @@ export function LinkOnlyInternalAnimations({
   border = { top: false, bottom: false },
   children,
   disabled = false,
+  skeleton = false,
+  lines = 1,
 }) {
   border = disabled ? { top: false, bottom: false } : border;
 
+  if (skeleton) {
+    return (
+      <LinkSkeleton dataCy={dataCy} className={className} lines={lines}>
+        {children}
+      </LinkSkeleton>
+    );
+  }
+
   return (
     <Link
       dataCy={dataCy}
-      className={cx(
-        animations.underlineContainer__only_internal_animations,
-        className
-      )}
+      className={cx(className, {
```

Comment:
OBS: 
Giver det nogensinde mening at have en animation på et link der er disabled? Er det ikke ret forvirrende for brugeren?
Jeg forslår at slå animationen fra, men I må lige sige til, hvad i tænker.

Reply:
Enig i at der ikke skal være animation når det er et disabled link.